app/livechat/client/views/app/livechatOfficeHours.js

```diff
@@ -156,17 +156,16 @@ Template.livechatOfficeHours.onCreated(function() {
 			open: new ReactiveVar(false),
 		},
 	};
+	this.officeHours = new ReactiveVar([]);
 
-	this.autorun(() => {
-		this.subscribe('livechat:officeHour');
-
-		if (this.subscriptionsReady()) {
-			LivechatOfficeHour.find().forEach(function(d) {
-				Template.instance().dayVars[d.day].start.set(moment.utc(d.start, 'HH:mm').local().format('HH:mm'));
-				Template.instance().dayVars[d.day].finish.set(moment.utc(d.finish, 'HH:mm').local().format('HH:mm'));
-				Template.instance().dayVars[d.day].open.set(d.open);
-			});
-		}
+	this.autorun(async () => {
```

Comment:
this doesn't need to be in a `autorun` anymore.. there is no reactivity anymore.